packages/ra-core/src/actions/accumulateActions.ts

```diff
@@ -6,16 +6,19 @@ export interface CrudGetManyAccumulateAction {
     readonly type: typeof CRUD_GET_MANY_ACCUMULATE;
     readonly payload: {
         resource: string;
-        ids: [];
+        ids: any[];
     };
     readonly meta: {
         accumulate: any;
     };
 }
 
+// Used to type the dispatcher function (the one injected by mapDispatchToProps)
+export type CrudGetManyAccumulate = (resource: string, ids: any[]) => void;
```

Comment:
Maybe this should be named `DispatchCrudGetManyAccumulate`. @fzaninotto, what do you think?

Reply:
"Dispatch" should only be used when the action creator is decorated by redux. Otherwise, we may end up with a `dispatchDispatchCrudGetManyAccumulate` prop name.

Reply:
Well, this is actually the type used for the function passed as prop by the `mapToDispatch` argument of connect

Reply:
understood, but it's not a dispatcher at that step

Reply:
Hu ? It is